### DISCUSSION TITLE
v23.0.7

### DISCUSSION THREAD

**Discussion by corhere** [moby/moby#46557] [Announcements]
## 23.0.7

For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:

- [docker/cli, 23.0.7 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A23.0.7)
- [moby/moby, 23.0.7 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A23.0.7)

### Bug Fixes and Enhancements

* Fix reloading the `insecure-registries` daemon configuration. https://github.com/moby/moby/pull/45572
* Allow empty body for `POST /commit` again. https://github.com/moby/moby/pull/45569
* Fix an issue which prevented encrypted overlay networks from functioning when the Swarm data path port is not set to 4789. https://github.com/moby/moby/pull/45638
* Fix an issue with graceful container shutdown. https://github.com/moby/moby/pull/45775
* Fix host-gateway support in `docker build`. https://github.com/moby/moby/pull/45791
* Fix missing Topology in Swarm cluster volume `NodeCSIInfo`. https://github.com/moby/moby/pull/45809
* seccomp: always allow `name_to_handle_at(2)`. https://github.com/moby/moby/pull/45834
* Fix an issue which prevented volumes mounted to a live-restored container from being removed. https://github.com/moby/moby/pull/45825
* client: resolve an incompatibility with Go 1.20.6, Go 1.20.7, Go 1.19.11 and Go 1.19.12. https://github.com/moby/moby/pull/45971
* Fix an issue which prevented process capabilities from being retained when starting a container as a non-root user with `--security-opt=no-new-privileges`. https://github.com/moby/moby/pull/46222
* Fixed a bug which caused named volumes that set custom `device` or `type` volume option to be unmounted when restarting the daemon and not live-restoring it properly. https://github.com/moby/moby/pull/46367
* windows: fix `--register-service` when executed from within binary directory. https://github.com/moby/moby/pull/46216
* Fix dockerd-rootless-setuptools.sh when user name contains a backslash. https://github.com/moby/moby/pull/46408

### Packaging Updates
* Update Go to 1.20.7. https://github.com/moby/moby/pull/46141
* Update containerd to v1.6.22. https://github.com/moby/moby/pull/46104
* Update runc to v1.1.9. https://github.com/moby/moby/pull/46229
* Delete Upstart init scripts and clean up sysvinit. https://github.com/moby/moby/pull/46046

<hr /><em>This discussion was created from the release <a href='https://github.com/moby/moby/releases/tag/v23.0.7'>v23.0.7</a>.</em>

**Comment by pihug12** [comment]
The RPM files of this version are not published (https://download.docker.com/linux/centos/7/x86_64/stable/Packages/, https://download.docker.com/linux/centos/8/x86_64/stable/Packages/ and https://download.docker.com/linux/centos/9/x86_64/stable/Packages/). Is it an oversight?
